internal/guest/storage/devicemapper/targets.go

```diff
@@ -46,8 +46,9 @@ func CreateZeroSectorLinearTarget(ctx context.Context, devPath, devName string,
 // 0 417792 verity 1 /dev/sdb /dev/sdc 4096 4096 52224 1 sha256 2aa4f7b7b6...f4952060e8 762307f4bc8...d2a6b7595d8..
 // |    |     |    |     |     |        |    |    |    |    |              |                        |
 // start|     |    |  data_dev |  data_block | #blocks | hash_alg      root_digest                salt
-//     size   |  version    hash_dev         |     hash_offset
-//          target                       hash_block
+//
+//	size   |  version    hash_dev         |     hash_offset
+//	  target                       hash_block
```

Comment:
I feel like this entire table will now be formatted wrong 😅

Reply:
It is.. I think this was a new rule in 1.19, as I don't remember 1.18 gofmt doing this on save

Reply:
I also cannot figure out how to keep this in a sane format... It adds an extra newline everytime. If you use /* */ comments, it at least doesn't shift to the left

Reply:
I think its cause of the difference in leading spacing
Can you try:
```go 
// CreateVerityTarget creates a dm-verity target for a given device and returns created virtual block device path.
//
// Example verity target table:
//
//	  0 417792 verity 1 /dev/sdb /dev/sdc 4096 4096 52224 1 sha256 2aa4f7b7b6...f4952060e8 762307f4bc8...d2a6b7595d8..
//	  |   |      |    |     |        |    |     |     |   |    |              |                        |
//	start |      |    | data_dev     |    |     | #blocks | hash_alg      root_digest                salt
//	     size    |  version      hash_dev | hash_block_sz |
//	           target              data_block_sz      hash_offset
//
// See [dm-verity] for more information
//
// [dm-verity]: https://www.kernel.org/doc/html/latest/admin-guide/device-mapper/verity.html#construction-parameters
```

(the urls should be rendered properly in the future....)
@anmaxvl : is this good?

Reply:
I was just reading the gofmt changes in 1.19, it's quite a bit https://tip.golang.org/doc/comment. I'll try what you posted